PayPalDataCollector/src/main/java/com/braintreepayments/api/PayPalDataCollectorRequest.kt

```diff
@@ -0,0 +1,5 @@
+package com.braintreepayments.api
+
+data class PayPalDataCollectorRequest(
+    val hasUserLocationConsent: Boolean
```

Comment:
🧶 : thought about `isLocationDataCollectionConsentGrantedByUser: Boolean`. It's an obnoxiously long variable name, but leading with is leans in to some default Java / Kotlin interop behavior with [getters and setters](https://kotlinlang.org/docs/java-interop.html#getters-and-setters). Total take-or-leave, but wanted to point it out for good measure.

Reply:
That's a good call. I think in this case, there wouldn't be a setter since `hasUserLocationConsent` is defined as a `val` and a merchant wouldn't realistically need to get the value from the request object since they'd be passing it into the SDK. However, I could see merchants using the getter in a unit test.

I'm in favor of sticking with this naming since it aligns with the MXO and Magnes SDKs.

Reply:
Got it yeah that makes sense. The part without the setter is key I hadn't realized it at first 👍 .